### PR TITLE
ideals: Add "Cockpit is tested"

### PR DESCRIPTION
**Pull request by martinpitt**
This came up a few times, and last night I just remembered it again. So here it is :-)

### CODE REVIEW

File: ideals.html [hunk 1]
```diff
@@ -151,6 +151,12 @@
 	<p>Cockpit allows others to add additional <span class="caps">UI</span> pieces. We do
 	not allow arbitrary extension of the <span class="caps">UI</span> all over the place, but
 	at distinct extension&nbsp;points.</p>
+
+	<h2>Cockpit is&nbsp;tested</h2>
+	<p>Every feature and change in Cockpit is covered by an automatic test.
+	A proposed change only lands in the code base after it passes all
+	tests. Tests run on all Operating System releases on which Cockpit is
```
Comment (dperpeet) [inline ideals.html]: These "all OS releases" could be a link to the actual page.

File: ideals.html [hunk 2]
```diff
@@ -151,6 +151,12 @@
 	<p>Cockpit allows others to add additional <span class="caps">UI</span> pieces. We do
 	not allow arbitrary extension of the <span class="caps">UI</span> all over the place, but
 	at distinct extension&nbsp;points.</p>
+
+	<h2>Cockpit is&nbsp;tested</h2>
+	<p>Every feature and change in Cockpit is covered by an automatic test.
```
Comment (dperpeet) [inline ideals.html]: Do we meet those strict criteria? "Cockpit features and changes are covered by automatic tests."?

File: ideals.html (hunk 1 above)
Comment (martinpitt) [inline ideals.html]: I made "all OS" a link to http://cockpit-project.org/running.html, but we don't have an easily accessible per-release list (at most in our bots/tests-scan, but I don't think we want to link to that).

File: ideals.html (hunk 2 above)
Comment (martinpitt) [inline ideals.html]: We certainly are far from 100% coverage, but in the spirit of ["Ideals are like stars..."](https://www.brainyquote.com/quotes/quotes/c/carlschurz389236.html) we generally try to abide by it and reject new features/fixes without tests.

Comment (stefwalter) [inline ideals.html]: I think this is good. Features that are not tested are basically proof of concepts, or failures. They rot and don't last. In broad terms: anything in Cockpit which hasn't rotted has some form of test covering it.

**Pull request merged**